src/EnergyPlus/StandardRatings.cc

```diff
@@ -4982,6 +4966,32 @@ namespace StandardRatings {
         }
     }
 
+    Real64 CondenserEnteringFluidTemperature(EnergyPlusData &state, DataPlant::CondenserType const CondenserType, Real64 LoadRatio)
+    {
+
+        Real64 CondenserEnteringFluidTemp = 0.0;
+        if (CondenserType == DataPlant::CondenserType::WaterCooled) {
+            // get the condenser entering water temperature for a given part load ratio in deg C
+            Real64 enteringWaterTemp = 18.33;
+            if (LoadRatio > 0.50) {
+                enteringWaterTemp = 7.22 + 22.22 * LoadRatio;
+            }
+            CondenserEnteringFluidTemp = enteringWaterTemp;
+        } else if (CondenserType == DataPlant::CondenserType::AirCooled) {
+            // get the outdoor air dry bulb temperature for a given part load ratio in deg C
+            Real64 enteringAirDBTemp = 12.78;
+            if (LoadRatio > 0.33) {
+                enteringAirDBTemp = 1.67 + 33.33 * LoadRatio;
+            }
+            CondenserEnteringFluidTemp = enteringAirDBTemp;
+        } else { // EvaporativelyCooled Condenser
+            // get the outdoor air wet bulb temperature for a given part load ratio in deg C
+            Real64 enteringAirWBTemp = 10.0 + 13.89 * LoadRatio;
+            CondenserEnteringFluidTemp = enteringAirWBTemp;
+        }
+        return CondenserEnteringFluidTemp;
+    }
+
```

Comment:
New refactored function for calculating the condenser entering fluid temperatures for the three condenser types of electric chiller models.